src/ConsoleContactAppManager/Validators.cs

```diff
@@ -0,0 +1,122 @@
+﻿// <copyright file="Validators.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace ContactManagerApp
+{
+    using System.Text.RegularExpressions;
+
+    /// <summary>
+    /// Class validators have the methods to check if the fields enetered by user are valid.
+    /// </summary>
+    internal static class Validators
+    {
+        /// <summary>
+        /// The CheckIfEmpty method checks if the contact list is empty.
+        /// </summary>
+        /// <param name="contactList"> The contact list is passed as input </param>
+        /// <returns>Returns 0 if there is no contact in the list and 1 if the list has contacts</returns>
+        public static int CheckIfEmpty(List<Contact> contactList)
+        {
+            if (contactList.Count == 0)
+            {
+                Console.WriteLine("The contact list is empty");
+                return 0;
+            }
+            else
+            {
+                return 1;
+            }
+        }
+
+        /// <summary>
+        /// Validates the email
+        /// </summary>
+        /// <param name="email"> The email entered by user is validated. </param>
+        /// <returns>Returns 0 if email is invalid and 1 if emailis valid.</returns>
+        public static int EmailValidate(string email)
+        {
+            string emailpattern = "^(\\w+)@([\\w\\-]+)((\\.(\\w){2,3})+)$";
+            if (!Regex.IsMatch(email, emailpattern))
+            {
+                Console.WriteLine("Enter a valid Email");
+                return 0;
+            }
+            else
+            {
+                return 1;
+            }
+        }
+
+        /// <summary>
+        /// GetInput method gets input and converts it into integer.
+        /// </summary>
+        /// <returns>integer</returns>
+        public static int GetInput()
```

Comment:
This method is not a validator method right so it should be present under an appropriate file.

Reply:
It is moved to InputHelpers file